letsdrill.rb

```diff
@@ -1,19 +1,28 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
-
+  case integer
+  when 91..100
+    "A"
+  when 81..90
+    "B"
+  when 71..80
+    "C"
+  when 61..70
+    "D"
+  when integer<60
+    "F"
+  else
+    "Invalid Input"
+  end
```

Comment:
Nice, clean solution.  

Tackling the invalid inputs is an extra step, but let's talk about the best way to do that.  This might be a good case for using a guard clause up front - something like `return 'Invalid Input' unless integer.is_a? Integer`.  But that gets me what the desired behavior would actually be here.  Obviously, this all depends on what sort of broader program this method would appear in, but my hunch is that in most use cases, either returning `nil` or raising a `TypeError` would be more helpful than returning a different string.